services/121-service/src/programs/program-attachments/program-attachments.controller.ts

```diff
@@ -78,6 +74,19 @@ export class ProgramAttachmentsController {
     @Param('programId', ParseIntPipe) programId: number,
     @Req() req: ScopedUserRequest,
   ): Promise<CreateProgramAttachmentResponseDto> {
+    const allowedMimeTypes = [
+      'image/jpeg',
+      'image/png',
+      'image/gif',
+      'image/webp',
+      'application/msword',
+      'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+      'application/pdf',
+    ];
+    if (!allowedMimeTypes.includes(file.mimetype)) {
+      throw new BadRequestException('Something went wrong: Validation failed');
```

Comment:
I thought we had a guideline to "only throw (generic) `Error`s"; (I think @RubenGeo had dealt with that at some point)

But it _does make sense_, to me at least, to throw something specific like this _in a controller_...